recipe/meta.yaml

```
@@ -1,67 +1,66 @@
-{% set libmamba_version = "0.24.0" %}
-{% set libmambapy_version = "0.24.0" %}
-{% set mamba_version = "0.24.0" %}
-{% set release = "2022.05.31" %}
-
-
-# NOTE FOR FUTURE PACKAGE BUILDERS/MAINTAINERS:
-#
-#   Currently, this package builds the same source twice. In future,
-#   it should be modified to build it once and package each piece
-#   independently.
-
+{% set name = "mamba" %}
+{% set libmamba_version = "1.0.0" %}
+{% set libmambapy_version = "1.0.0" %}
+{% set mamba_version = "1.0.0" %}
+{% set release = "2022.11.01" %}
+{% set build_mamba = "false" %}
 
 package:
-  name: libmambapy-split
+  name: mamba-split
 
 source:
-  url: https://github.com/mamba-org/mamba/archive/{{ release }}.tar.gz
-  sha256: 7c63fcd10371dca78e2715826fb6c18e35f51d947faf40a728854dabfb4caab8
+  url: https://github.com/mamba-org/mamba/archive/refs/tags/{{ release }}.tar.gz
+  sha256: f77b858012d3a93b75565c60ab7594c9508bed559b8369aca32f736e9186a733
   patches:
-    # Fix some compilation issues.
-    - patches/libmamba_CMakeLists.txt.patch
-    - patches/libmamba_src_core_context.cpp.patch
+    - add_find_package_fmt.patch
 
 build:
   number: 0
 
 requirements:
   build:
-    - patch     # [unix]
+    - patch     # [not win]
     - m2-patch  # [win]
 
 outputs:
   - name: libmamba
     version: {{ libmamba_version }}
-    script: build_libmamba.sh                     # [unix]
-    script: bld_libmamba.bat                      # [win]
+    script: build_mamba.sh                       # [unix]
+    script: build_mamba.bat                      # [win]
     build:
       run_exports:
         - {{ pin_subpackage('libmamba', max_pin='x.x') }}
+      ignore_run_exports:
+        - spdlog
+        - python    # [win]
+        - libiconv  # [osx]
       missing_dso_whitelist:  # [s390x]
         - $RPATH/ld64.so.1    # [s390x] Known s390x `ld64.so` issue.
-      ignore_run_exports:
-        - spdlog-fmt-embed    # Only using headers
     requirements:
       build:
         - {{ compiler('cxx') }}
         - cmake
-        - ninja-base
+        - ninja
+        - python  # [win]
       host:
-        - libsolv >=0.7.19
+        - libsolv 0.7.22
         - libcurl
         - openssl
         - libarchive
         - nlohmann_json
-        - cpp-filesystem >=1.5.8
         - cpp-expected
-        - reproc >=14.2.1
-        - reproc-cpp >=14.2.1
-        - spdlog-fmt-embed
+        - reproc-cpp 14.2.4
+        - spdlog
         - yaml-cpp
         - termcolor-cpp
         - cli11
+        - fmt
+        - libiconv                               # [osx]
         - winreg                                 # [win]
+      run:
+        - libsolv >=0.7.19
```

Shouldn't this be >=0.7.22 (as in host)?

```suggestion
        - libsolv >=0.7.22
```